collector_client_http.go

```diff
@@ -10,8 +10,8 @@ import (
 	"net/url"
 	"time"
 
-	"github.com/golang/protobuf/proto"
-	"github.com/lightstep/lightstep-tracer-common/golang/protobuf/collectorpb"
+	"github.com/gogo/protobuf/proto"
```

Comment:
Now that we are importing gogo directly, we should add a `constraint` to the Gopkg.toml file on the version.

Reply:
can't add a constraint on gogo because there is no go source in the root of this repo. 🤷‍♂️

Reply:
```[[constraint]]
  name = "github.com/gogo/protobuf"
  version = "1.2.1"``` does that not work?

Reply:
IT works now. I'm not sure what sort of state I was in, because it wasn't working, because there's no source in the root of that repo. I do not like `dep`.

Reply:
(Fixed)